GETTING_STARTED.md

```diff
@@ -0,0 +1,602 @@
+  - name: centos-7.2
+```
+
+To this:
+
+```
+platforms:
+  - name: ubuntu
+```
+
+So, right now, you .kitchen.yml should look like this:
+
+.kitchen.yml
+```
+---
+driver:
+  name: terraform
+
+provisioner:
+  name: terraform
+  variable_files:
+    - testing.tfvars
+
+platforms:
+  - name: ubuntu
+
+suites:
+  - name: default
+    run_list:
+    attributes:
+```
+
+Next, let's add a new section - the transport section.  This is what will allow kitchen-terraform to ssh into your terraform'd instances and run your inspec tests.  You will need to include the path to the private key half of your AWS keypair - wherever you keep it on your workstation.
+
+```
+transport:
+  ssh_key: ~/path/to/your/private/aws/key.pem
+```
+
+And now let's add another section, the verifier section.  The verifier is what will verify whether your test kitchen instances match what is laid out in your inspec files.
+
+```
+verifier:
+  name: terraform
+  format: doc
+  groups:
+  - name: inspec
+    tests:
+      -default
+    hostnames: public_dns
+    username: ubuntu
+```
+
+And let's stop and talk about what we did here - as this is where you will see some of the uniqueness of kitchen-terraform.
+
+With these lines:
+
+```
+verifier:
+  name: terraform
+  format: doc
+```
+
+We have specified that we are using the terraform verifier (which is included with kitchen terraform).  We are also using the format type of doc for our tests' output.
+
+And we have also specified a group of tests.
+
+```
+  groups:
+  - name: inspec
+    tests:
+      -default
+    hostnames: public_dns
+    username: ubuntu
+```
+
+Our group's name is inspec, and we expect to find a test file called default_spec.rb within that group. So this .kitchen.yml expects to find a directory called "inspec", with a file called "default_spec.rb" within that group
+
+Additionally, kitchen-terraform also needs to know what hostnames it needs to ssh into to run our specs.  We have specified "public_dns" - which is an output value we will need to add to output.tf in a bit.
+
+Finally, there is the username - this is the name kitchen terraform will use to ssh into the hostnames with.  In this tutorial we are using ubuntu instances and the default username is ubuntu.
+
+
+At this point, your .kitchen.yml file should look like this:
+
+.kitchen.yml
+```
+---
+driver:
+  name: terraform
+
+provisioner:
+  name: terraform
+  variable_files:
+    - testing.tfvars
+
+platforms:
+  - name: ubuntu
+
+suites:
+  - name: default
+    run_list:
+    attributes:
+
+transport:
+  ssh_key: ~/path/to/your/private/aws/key.pem
+
+verifier:
+  name: terraform
+  format: doc
+  groups:
+  - name: inspec
+    tests:
+      - default
+    hostnames: public_dns
+    username: ubuntu
+```
+
+Go head and save and close the file.
+
+## Writing a test
+
+Now, we have some work to do on our workstation.  First, let's set up an inspec directory within our test directories
+
+```
+  $ mkdir test/integration/default/inspec
```

Comment:
The "legacy mode" approach isn't supported by the plugin. The contents of `default` should be `inspec.yml` and `controls/`.